tasks/load_instackenv.yml

```diff
@@ -37,6 +37,10 @@
       set_fact:
         undercloud_hostname: "{{ undercloud_hostname|default(instackenv_content.nodes[0].pm_addr | replace('mgmt-','') | replace('-drac', '')) }}"
 
+    - name: for machine type
```

Comment:
why is this needed?

Reply:
because used machine type is fine if virtual_uc is false